<ui version="4.0" >
 <class>EditRemote</class>
 <widget class="QDialog" name="EditRemote" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>369</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAbout" >
      <attribute name="title" >
       <string>About</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="lblName" >
         <property name="text" >
          <string>Name:</string>
         </property>
         <property name="buddy" >
          <cstring>txtname</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="txtname" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblComment" >
         <property name="text" >
          <string>Comment:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblAuthors" >
         <property name="text" >
          <string>Authors:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="lblIcon" >
         <property name="text" >
          <string>Icon:</string>
         </property>
         <property name="buddy" >
          <cstring>txticon</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="txticon" />
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="btnSearch" >
         <property name="text" >
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label" >
         <property name="toolTip" >
          <string>Target receiver suggestions</string>
         </property>
         <property name="text" >
          <string>For Receivers:</string>
         </property>
         <property name="buddy" >
          <cstring>txtCompatible</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2" >
        <widget class="QLineEdit" name="txtCompatible" />
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QPlainTextEdit" name="txtcomment" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QPlainTextEdit" name="txtauthors" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSettings" >
      <attribute name="title" >
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QTableView" name="settingsView" />
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btnSettingsAdd" >
           <property name="contextMenuPolicy" >
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="text" >
            <string>Add</string>
           </property>
           <property name="popupMode" >
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnSettingsRemove" >
           <property name="text" >
            <string>Remove</string>
           </property>
           <property name="popupMode" >
            <enum>QToolButton::MenuButtonPopup</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabKeys" >
      <attribute name="title" >
       <string>Keys</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QTableView" name="keysView" />
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" >
           <item>
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="btnKeyAdd" >
             <property name="contextMenuPolicy" >
              <enum>Qt::ActionsContextMenu</enum>
             </property>
             <property name="text" >
              <string>Add</string>
             </property>
             <property name="popupMode" >
              <enum>QToolButton::DelayedPopup</enum>
             </property>
             <property name="toolButtonStyle" >
              <enum>Qt::ToolButtonIconOnly</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnKeyRemove" >
             <property name="text" >
              <string>Remove</string>
             </property>
             <property name="popupMode" >
              <enum>QToolButton::MenuButtonPopup</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Events from receivers</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" >
          <item>
           <widget class="QListView" name="receiversView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Press a key on your remote to add keys to / modify keys of the right list.</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="lblStatus" >
       <property name="text" >
        <string>Ready</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>txtname</tabstop>
  <tabstop>txtcomment</tabstop>
  <tabstop>txtauthors</tabstop>
  <tabstop>txticon</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>txtCompatible</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>settingsView</tabstop>
  <tabstop>keysView</tabstop>
  <tabstop>receiversView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
